### PR TITLE
james-doug-liz 

### PR DESCRIPTION
**Pull request by ghost**
#### Single-line Summary

**Today, James, Doud and Elizabeth paired together. It took about 2**
#### Reflect and summarize on your process for each `TODO` item :
1. First, we started on Including the required files
2. Next, we setup the routers linking urls to controllers 
3. Then we built out controllers
4. Then we deleted redundant code
#### Checklist (before submitting, fill in each set of square brackets with an 'x')
- [x] We have titled the Pull Request similar to our branch name (ex: 'brian-rick').
- [x] This PR includes commits from both myself and my partner; e.g. We followed good pair programming practices by switching driver/navigator roles.
- [x] There is no extraneous, unrelated code included in this PR.
- [x] We have summarized our `TODO:` process above.

### CODE REVIEW

File: starter-code/scripts/controllers/routes.js [hunk 1]
```diff
@@ -1,6 +1,22 @@
...
-/* TODO: What function do we call to activate page.js?
+$(document).ready(function(){
+	page();
+});
+page('/' , article);
+page('/about', about);
+
+function article() {
+  articleController.reveal();
+  console.log('article() ran');
```
Comment (philmein23) [inline starter-code/scripts/controllers/routes.js:13]: It's great to console log to verify if the target has been hit. Try to remove all console logs before you submit the assignment (or before submitting to production).

File: starter-code/scripts/controllers/articleController.js [hunk 2]
```diff
@@ -2,7 +2,11 @@
   var articleController = {};
 
   articleController.reveal = function() {
-    /* TODO: Reveal only the articles section! */
+
+    $('.tab-content').hide();
+
```
Comment (nhuntwalker) [inline starter-code/scripts/controllers/articleController.js:7]: unnecessary whitespace

File: starter-code/scripts/controllers/aboutController.js [hunk 3]
```diff
@@ -2,7 +2,11 @@
   var aboutController = {};
 
   aboutController.reveal = function() {
-    /* TODO: Reveal only the about section! */
+    /* TODO DONE: Reveal only the about section! */
+    $('.tab-content').hide();
+
```
Comment (nhuntwalker) [inline starter-code/scripts/controllers/aboutController.js:7]: unnecessary whitespace
